metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "nmaludy-erlang",
+  "name": "puppet-erlang",
   "version": "0.1.0",
-  "author": "nmaludy",
+  "author": "puppet",
```

Comment:
I think this should be “Vox Pupuli” maybe

Reply:
```"author": "Vox Pupuli",```